UefiCpuPkg/PiSmmCpuDxeSmm/PiSmmCpuDxeSmm.inf

```diff
@@ -103,8 +103,8 @@
   CpuPageTableLib
   MmSaveStateLib
   SmmCpuSyncLib
+  PanicLib

```

Comment:
Did `PanicLib` intentionally drop the `MU_CHANGE` tag? It is not in https://github.com/tianocore/edk2/blob/master/UefiCpuPkg/PiSmmCpuDxeSmm/PiSmmCpuDxeSmm.inf.

Reply:
That was accidental, I have added it back, thanks.